app/controllers/submit.js

```diff
@@ -688,6 +691,9 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
                     label: `Preprints - ${this.get('editMode') ? 'Edit' : 'Submit'} - Save and Continue Basics Section`
                 });
             // Saves the description/tags on the node and the DOI on the preprint, then advances to next panel
+            if (!this.get('basicsValid')) {
```

Comment:
Confirmed that this is the only section where a user will be hitting enter in a single-line text field, so none of the other validators need to be checked. 

If we add fields to these forms in the future that may change, in which case "enter to submit" would need to be added to those fields later.